File: app/models/comic.rb
class Comic < ActiveRecord::Base
	scope :ordered, :order => 'date_published asc'
	scope :limited, :limit => 100

	attr_accessible :author, :characters, :contents, :date_display, :date_published, :img, :img_link, :img_thumb, :notes, :summary, :title, :transcript, :transcript_text
	serialize :characters
	serialize :contents

end

File: public/javascripts/models/comic.js

// COMIC.JS

var Comic = Backbone.Model.extend({
	defaults: {
		"title": "",
		"author": "",
		"date_display": "",
		"date_published": "",
		"summary": "",
		"characters": "",
		"contents": "",
		"notes": "",
		"transcript": "",
		"transcript_text": "",
		"img": "",
		"img_thumb": "",
		"img_link": "",
		"comic_id": ""
	}
	// url: function(){
	// 	var base = '/gallery';
	// 	if (this.isNew()) return base;
	// 	return base + (base.charAt(base.length - 1) == '/' ? '' : '/') + this.id;
	// }
});